datalad/distribution/create_publication_target_sshwebserver.py

```diff
@@ -43,78 +43,78 @@ class CreatePublicationTargetSSHWebserver(Interface):
         # TODO: Figure out, whether (and when) to use `sshurl` as push url
         dataset=Parameter(
             args=("--dataset", "-d",),
-            doc="""specify the dataset to create the publication target for. If
+            doc="""specify the dataset to create the publication target for.  If
                 no dataset is given, an attempt is made to identify the dataset
                 based on the current working directory""",
             constraints=EnsureDataset() | EnsureNone()),
         sshurl=Parameter(
             args=("sshurl",),
             doc="""SSH URL to use to log into the server and create the target
-                dataset(s). This also serves as a default for the URL to be
+                dataset(s).  This also serves as a default for the URL to be
                 used to add the target as a sibling to `dataset` and as a
                 default for the directory on the server, where to create the
-                dataset.""",
+                dataset""",
             constraints=EnsureStr()),
         target=Parameter(
             args=('target',),
-            doc="""Sibling name to create for this publication target.
+            doc="""sibling name to create for this publication target.
                 If `recursive` is set, the same name will be used to address
-                the subdatasets' siblings. Note, that this is just a
+                the subdatasets' siblings.  Note, that this is just a
                 convenience function, calling add_sibling after the actual
-                creation of the target dataset(s). Whenever the creation fails,
-                no siblings are added.""",
+                creation of the target dataset(s).  Whenever the creation fails,
+                no siblings are added""",
             constraints=EnsureStr() | EnsureNone(),
             nargs="?"),
         target_dir=Parameter(
             args=('--target-dir',),
-            doc="""Path to the directory on the server where to create the
-                dataset. By default it's wherever `sshurl` points to. If a
+            doc="""path to the directory on the server where to create the
+                dataset.  By default it's wherever `sshurl` points to.  If a
                 relative path is provided, it's interpreted as relative to the
                 user's home directory on the server.
                 Especially when using `recursive`, it's possible to provide a
                 template for building the URLs of all (sub)datasets to be
-                created by using placeholders. If you don't provide a template
+                created by using placeholders.  If you don't provide a template
                 the local hierarchy with respect to `dataset` will be
                 replicated on the server rooting in `target_dir`.\n
-                List of currently available placeholders:\n
+                list of currently available placeholders:\n
```

Comment:
That is the other new sentence withing the paragraph so should stay capitalized.